io.github.am2rcommunitydevelopers.AM2RLauncher.yaml

```diff
@@ -0,0 +1,134 @@
+app-id: io.github.am2rcommunitydevelopers.AM2RLauncher
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet6
+  - org.freedesktop.Sdk.Extension.openjdk11
+  - org.gnome.Sdk.Compat.i386
+
+finish-args:
+# TODO: crashes with wayland on filepicker and font being whacky. Likely Eto bug, test and report there.
+- --socket=x11
+- --share=ipc
+
+# Network needed for updates, pulseaudio for sound
+- --share=network
+- --socket=pulseaudio
+
+# Home for getting the 1.1 zip, config because of profiles having their saves there, apktool for apk creation
+- --filesystem=home:ro
+- --filesystem=~/.config/
+- --filesystem=xdg-data/apktool/:create
+
+# libappindicator stuff
+- --talk-name=com.canonical.indicator.application
+- --talk-name=org.freedesktop.Notifications
+- --talk-name=org.kde.StatusNotifierWatcher
+
+# Fuse / Appimage related
+- --talk-name=org.freedesktop.Flatpak.*
+- --device=all
+- --filesystem=/tmp
+
+# Java and x32 stuff
+- --env=JAVA_HOME=/app/jre
+- --env=PATH=/usr/bin:/app/bin:/app/jre/bin
+- --env=XDG_CONFIG_DIRS=/etc/xdg:/usr/lib/x86_64-linux-gnu/GL:/usr/lib/i386-linux-gnu/GL
```

Comment:
what is this for? I doubt it does what you think it does.

Reply:
The XDG config dirs are necessary in order for the 32-bit stuff to run.
Java home and including that into path is necessary for java to work.

Reply:
Really? Are you sure?